Source/Csla/Serialization/Mobile/CustomSerializers/PocoSerializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ClaimsPrincipalFormatter.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>A formatter that can serialize and deserialize a POCO</summary>
+//-----------------------------------------------------------------------
+
+using System.Text.Json;
+
+namespace Csla.Serialization.Mobile.CustomSerializers;
+
+/// <summary>
+/// A formatter that can serialize and deserialize a POCO
+/// by using System.Text.Json.
+/// </summary>
+public class PocoSerializer<T> : IMobileSerializer
+{
+  /// <inheritdoc />
+  public static bool CanSerialize(Type type) => type == typeof(T);
+
+  /// <inheritdoc />
+  public object Deserialize(SerializationInfo info)
+  {
+    var json = info.GetValue<string>("s");
```

Comment:
I'm not that deep in the (de-)serialization process but the same key is used by the `ClaimsPrincipalSerializer`. Is that correct?

Reply:
Yes, but the key name is per-instance of a serialized thing. There's no need for keys to be unique across different instances of serialized objects.